### PR TITLE
Map locations

### PR DESCRIPTION
**Pull request by fingoltin**

### CODE REVIEW

File: src/main/java/org/miracum/etl/fhirtoomop/model/omop/Concept.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public String getLabel() {
   }
 
   public enum Vocabulary {
-    ICD10GM("ICD10GM"),
+    ICD10GM("ICD10"),
```
Comment (michealharrington) [inline src/main/java/org/miracum/etl/fhirtoomop/model/omop/Concept.java]: Should the `enum` "key" change to match?

**Comment by michealharrington** [comment]
Should we have separate PRs for Location and ICD-10 mapping?

**Comment by fingoltin** [comment]
> Should we have separate PRs for Location and ICD-10 mapping?

Probably yes.

File: src/main/java/org/miracum/etl/fhirtoomop/model/omop/Concept.java (hunk 1 above)
Comment (fingoltin) [reply src/main/java/org/miracum/etl/fhirtoomop/model/omop/Concept.java]: Will be in the "vocabulary" update PR.

**Review by michealharrington — APPROVED** [review]

**Review by robinmohlePOP — APPROVED** [review]

**Pull request merged**